src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/Frame.cs

```diff
@@ -75,6 +75,8 @@ public abstract partial class Frame : IFrameControl
         protected readonly long _keepAliveMilliseconds;
         private readonly long _requestHeadersTimeoutMilliseconds;
 
+        private int _responseBytesWritten;
```

Comment:
I'll use it when fixing https://github.com/aspnet/KestrelHttpServer/issues/175, so less diff when I make that change :smile: